packages/cli/plugin-storybook/template/user-preview.tmpl

```diff
@@ -1,4 +1,4 @@
-const userPreview = require('<%= userPreviewPath %>');
+import * as userPreview from '<%= userPreviewPath %>';
```

Comment:
why need to change to esm, what happened?

Reply:
Storybook doesn't seem to be able to handle preview files in cjs format